good-citizen.md

```diff
@@ -0,0 +1,50 @@
+# Being a Good Citizen of Exercism 
+
+We want Exercism to be a great place to spend time, whoever you are.
+To ensure that Exercism is that, we place a strong focus on being good community citizens.
+This is formalised in our [Code of Conduct](https://exercism.io/code-of-conduct) but can be summed up simply in one sentence: **Be kind.**
+
+## The words we use
+
+There are lots of ways to embrace being kind within Exercism, but the most powerful is to consider how your words affect others.
+
+Words in themselves are meaningless. 
+They are vibrations caused by puffs of air coming out of our lungs.
+However, because of **context**, words are powerful.
+
+It's easy to know how you intend your words to come across, but it's difficult to know how others hear your words.
+What we say is strongly tempered by our experiences in life, our cultural norms, and our primary spoken language.
```

Comment:
Can we use "native language" instead of spoken language? For people in the Deaf community their native language isn't spoken.